device_backends/xdma/src/XdmaBackend.cc

```diff
@@ -39,15 +39,8 @@ namespace ChimeraTK {
       }
     }
 
-    // (Re-)Open the event files which are needed
+    // (Re-)Open the event files only when you acquire an accessaor.
     std::for_each(_eventFiles.begin(), _eventFiles.end(), [](auto& eventFile) { eventFile = nullptr; });
-    for(size_t i = 0; i < _maxInterrupts; i++) {
-      if(_startInterruptHandlingCalled[i]) {
```

Comment:
With this code gone, the variable _startInterruptHandlingCalled is obsolete. Please remove it.